src/test/jtx/impl/mpt.cpp

```diff
@@ -161,7 +161,24 @@ MPTTester::authorize(MPTAuthorize const& arg)
     }
     if (arg.holder)
         jv[sfMPTokenHolder.jsonName] = arg.holder->human();
-    submit(arg, jv);
+    if (submit(arg, jv) == tesSUCCESS)
+    {
+        if (arg.account == nullptr || *arg.account == issuer_)
```

Comment:
whats the need for `arg.account == nullptr` check here?

Reply:
Nevermind, the default account is the issuer.

Reply:
It's valid for `arg.account` to be nullptr. If `arg.account` is nullptr then by default it's the `issuer_`.